src/style.css

```diff
@@ -1,3 +1,78 @@
 body {
+  font-family: Roboto, system-ui;
```

Comment:
Consider adding fallback font families in case 'Roboto' is not available. This ensures text is still legible with a similar font. Example: font-family: Roboto, Arial, sans-serif;